services/beaconblockproposer/standard/propose.go

```diff
@@ -334,13 +236,10 @@ func (*Service) confirmProposalData(_ context.Context,
 		return errors.New("proposal data for incorrect slot")
 	}
 
-	proposalGraffiti, err := proposal.Graffiti()
-	if err != nil {
-		return errors.Wrap(err, "failed to obtain proposal graffiti")
-	}
-	if !bytes.Equal(proposalGraffiti[:], graffiti[:]) {
-		return errors.New("proposal data contains incorrect graffiti")
-	}
+	// RANDAO reveal can be different in DVT situations, so do not check it.  It wil have already been checked by the underlying
```

Comment:
Small typo

```suggestion
	// RANDAO reveal can be different in DVT situations, so do not check it. It will have already been checked by the underlying
```